image_tools/conf.py

```diff
@@ -235,6 +235,10 @@
                 "python": "39",
                 "java": "11",
                 "hadoop_short_version": "3.2",
+                "hadoop_long_version": "3.3.1",
+                "aws_java_sdk_bundle": "1.11.901",
+                "azure_storage": "7.0.1",
+                "azure_keyvault_core": "1.0.0",
```

Comment:
Because of this missing jars the `pyspark-k8s:3.2.1-stackable23.4` version should be broken (at least the s3 and azure stuff). This PR fixes this by letting curl check the response code